dev-utils/test-config.js

```diff
@@ -172,7 +172,8 @@ function getBrowserList(pkg = 'default') {
       },
       {
         browserName: 'firefox',
-        browserVersion: 'latest',
+        // beware that if we update to 99 or more we will need to cope with https://github.com/karma-runner/karma-sauce-launcher/issues/275
+        browserVersion: '98',
```

Comment:
for now, avoid investing time in this annoying issue where saucelabs, webdriverio, and, karma-sauce-launcher integration are causing an error.

<img width="615" alt="screenshot" src="https://github.com/elastic/apm-agent-rum-js/assets/15065076/02592330-e305-4b46-ad00-cdc9cbc3fc5f">

Related info:

https://github.com/karma-runner/karma-sauce-launcher/issues/275
https://github.com/elastic/apm-agent-rum-js/pull/1303